.github/workflows/pr_build.yml

```diff
@@ -73,12 +78,8 @@ jobs:
         os: [macos-13]
         java_version: [8, 11, 17]
         test-target: [rust, java]
-        is_push_event:
-          - ${{ github.event_name == 'push' }}
-        exclude: # exclude java 11 for pull_request event
-          - java_version: 11
-            is_push_event: false
       fail-fast: false
+    if: github.event_name == 'push'
```

Comment:
Maybe we can consider to split tests to several pipelines later so each one won't take too long time. Then we probably can have MacOS pipeline in pull request.

Reply:
Yea, I plan to split shuffle tests into a separate job later. But I think we can still keep the MacOS pipelines as post-commit because later on we need to add more pipelines for Spark SQL tests.

Reply:
yea I think so too